README.md

```diff
@@ -0,0 +1,33 @@
+# vpphelper
+
+vpphelper provides helper functions (using govpp) that helps to connect to VPP and configure VPP using a configuration (.conf) file.
+
+* There is an `option` struct defined that has two fields:  
```

Comment:
variables and functions starting with lower case letters are package private, and not available to library users.
So 'option' isn't a thing they can access.

Document instead the `With*` functions that are passed into the constructor.